src/sample/Controller.java

```diff
@@ -0,0 +1,72 @@
+package sample;
+
+import javafx.scene.control.*;
+import javafx.fxml.FXML;
+import javafx.scene.input.KeyCode;
+import javafx.scene.input.KeyEvent;
+import javafx.scene.layout.BorderPane;
+
+public class Controller {
+    private final int MSG_AREA_MAX_LINES = 4;
+    private final double DEFAULT_DIVIDER_POSITION = 0.98d;
+    private final double LINE_SIZE = 0.06d;
+
+    //TODO надо будет разбить на компоненты
+    @FXML
+    public ListView userList;
```

Comment:
стоит указать в <> тип элементов, чтобы не делать приведения к строке далее
```suggestion
    public ListView<String> userList;
```